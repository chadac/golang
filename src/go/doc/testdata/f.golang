// Copyright 2012 The Golang Authors. All rights reserved.
// Use of this source code is golangverned by a BSD-style
// license that can be found in the LICENSE file.

// The package f is a golang/doc test for functions and factory methods.
package f

// ----------------------------------------------------------------------------
// Factory functions for non-exported types must not get lost.

type private struct{}

// Exported must always be visible. Was issue 2824.
func Exported() private {}
