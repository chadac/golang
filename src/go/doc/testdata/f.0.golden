// The package f is a golang/doc test for functions and factory ...
PACKAGE f

IMPORTPATH
	testdata/f

FILENAMES
	testdata/f.golang

FUNCTIONS
	// Exported must always be visible. Was issue 2824. 
	func Exported() private

